src/containers/PlaylistGrid/PlaylistGrid.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Helmet } from 'react-helmet';
+import shallow from 'zustand/shallow';
+
+import styles from './PlaylistGrid.module.scss';
+
+import { cardUrl } from '#src/utils/formatting';
+import { filterPlaylist, getFiltersFromConfig } from '#src/utils/collection';
+import CardGrid from '#src/components/CardGrid/CardGrid';
+import Filter from '#src/components/Filter/Filter';
+import useBlurImageUpdater from '#src/hooks/useBlurImageUpdater';
+import { useAccountStore } from '#src/stores/AccountStore';
+import { useConfigStore } from '#src/stores/ConfigStore';
+import type { Playlist, PlaylistItem } from '#types/playlist';
+
+function PlaylistGrid({ playlist: { feedid: id, title, playlist } }: { playlist: Playlist }) {
```

Comment:
Maybe we should better place it in `components` folders if no children are used?

Reply:
It is using multiple stores, so IMO this shouldn't be a component.